docs/_includes/date-picker.html

```diff
@@ -0,0 +1,212 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+
+    <h3>Single Pane</h3>
+
+    <button class="btn" onclick="document.getElementById('date-picker-example').toggleAttribute('hidden')">Show/Hide</button>
+
+    <div id="date-picker-example" class="date-picker date-picker--single-month" role="dialog" aria-label="date picker">
```

Comment:
Typically in BEM we strive to make a block usable without any modifiers. If a modifiers is always required it can indicate that things need splitting out further. Can we make `date-picker--single-month` the sensible default without needing to specify it? I imagine you are going to have a `date-picker--multi-month` or the like?

Reply:
Right now the only change for single month is that these margins (shown in orange) are redundant and removed.
![image](https://user-images.githubusercontent.com/26027232/209207205-9bc79f2b-fb63-46f8-bd4c-35d1a6d59481.png)
I added the class instead of using a complicated CSS selector to account for potential future markup changes, but on second thought markup changes will need style changes either way. I'll use an interesting selector instead.